### PR TITLE
Add interactive CLI

### LINKED ISSUE #1: Interactive CLI
Would be nice to have an interactive CLI when it's run without arguments.

The user would be presented with a [list of process](https://github.com/sindresorhus/ps-list) and can easily go up and down then hit enter to kill one.

Can be implemented using [Inquirer.js](https://github.com/SBoudrias/Inquirer.js).

Similar example from `yo`:

screen shot 2013-10-19 at 1 44 14 am

-

Should look something like this:

```
❯ Process1  24241
  Process2  24242
  Process3  24243
```

With the number being the PID and `chalk.dim()`.
Sorted by PID.

### PR DESCRIPTION
**Pull request by kevva**
Initial implementation. This needs a nicely outputted list and maybe some colors.

Fixes #1.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Awesome! :)

Can you dim the PID according to the description in https://github.com/sindresorhus/fkill-cli/issues/1?

**Comment by kevva** [comment]
Yup, fixed! Also changed the sorting to PID instead of name.

File: cli.js [hunk 1]
```diff
@@ -15,9 +18,45 @@ var cli = meow({
 	]
 });
 
+function listProcesses(processes) {
+	inquirer.prompt([{
+		name: 'processes',
+		message: 'Running processes:',
+		type: 'list',
+		choices: processes.sort(function (a, b) {
+			return a.pid - b.pid;
```
Comment (sindresorhus) [inline cli.js]: [`num-sort`](https://github.com/sindresorhus/num-sort) :p

File: cli.js [hunk 2]
```diff
@@ -15,9 +18,45 @@ var cli = meow({
...
+		}).map(function (proc) {
+			return {
+				name: proc.name + '  ' + chalk.dim(proc.pid),
```
Comment (sindresorhus) [inline cli.js]: Actually, one space looks better than two.

**Comment by sindresorhus** [comment]
Awesome! It's out ;)

**Pull request merged**